pipelines/azure-pipelines-pr.yaml

```
@@ -0,0 +1,80 @@
+            allowTelemetryCollection: false
+        - task: tfsec@1
+          displayName: 'Terraform SEC check'
+          inputs:
+            version: '$(TERRAFORM_SEC_VERSION)'
+            dir: '$(System.DefaultWorkingDirectory)'
+        - script: |
+            cd /tmp
+            curl -sSLo /tmp/terraform-docs.tar.gz https://terraform-docs.io/dl/v0.19.0/terraform-docs-v0.19.0-$(uname)-amd64.tar.gz
+            tar -xzf /tmp/terraform-docs.tar.gz
+            chmod +x /tmp/terraform-docs
+          displayName: 'Download terraform-docs' 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)